### PR TITLE
fix bug in current nlp documentation

### PR DESCRIPTION
**Pull request by liususan091219**
## Why are these changes needed?

The nlp documentation currently has some formatting issues as in PR #752 . This PR fix these issues. 

## Checks

- [x] I've used [pre-commit](https://microsoft.github.io/FLAML/docs/Contribute#pre-commit) to lint the changes in this PR, or I've made sure [lint with flake8](https://github.com/microsoft/FLAML/blob/816a82a1155b4de4705b21a615ccdff67c6da379/.github/workflows/python-package.yml#L54-L59) output is two 0s.
- [x] I've included any doc changes needed for https://microsoft.github.io/FLAML/. See https://microsoft.github.io/FLAML/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

File: flaml/automl.py [hunk 1]
```diff
@@ -2376,7 +2376,7 @@ def custom_metric(
         def cv_score_agg_func(val_loss_folds, log_metrics_folds):
             return metric_to_minimize, metrics_to_log
         ```
-                “val_loss_folds” - list of floats, the loss scores of each fold; “log_metrics_folds” - list of dicts/floats, the metrics of each fold to log.
+            “val_loss_folds” - list of floats, the loss scores of each fold; “log_metrics_folds” - list of dicts/floats, the metrics of each fold to log.
```
Comment (sonichi) [inline flaml/automl.py:2379]: Not sure about this change. Are you able to compile the doc now?

**Review by skzhang1 — APPROVED** [review]

File: flaml/automl.py (hunk 1 above)
Comment (liususan091219) [reply flaml/automl.py]: image

Comment (liususan091219) [reply flaml/automl.py]: Please find the compiled website above.

Comment (sonichi) [reply flaml/automl.py]: "the following signature" -> "the following input arguments"?
"This function should" should not be in the same line as "log_metrics_folds".

**Review by sonichi — APPROVED** [review]

**Pull request merged**